scheduler/pkg/kafka/pipeline/multi_topic_consumer.go

```diff
@@ -56,13 +58,17 @@ func NewMultiTopicsKafkaConsumer(
 		requests: cmap.New(),
 		tracer:   tracer,
 	}
-	err := consumer.createConsumer()
+	err := consumer.createConsumer(logger)
 	return consumer, err
 }
 
-func (c *MultiTopicsKafkaConsumer) createConsumer() error {
+func (c *MultiTopicsKafkaConsumer) createConsumer(logger log.FieldLogger) error {
 	consumerConfig := kafka_config.CloneKafkaConfigMap(c.config.Consumer)
 	consumerConfig["group.id"] = c.id
+	consumerConfig["go.events.channel.enable"] = true
+	consumerConfig["go.application.rebalance.enable"] = true // Required for Events()
+	consumerConfig["partition.assignment.strategy"] = "roundrobin"
```

Comment:
Have you considered [stickyAssignor](https://kafka.apache.org/23/javadoc/org/apache/kafka/clients/consumer/StickyAssignor.html)? We should look into it more broadly

Reply:
Will leave it as `roundrobin` for now, since `sticky` give the following error:
```
            Unsupported partition.assignment.strategy: sticky
            {
                code: -186,
                str: "Unsupported partition.assignment.strategy: sticky",
                fatal: false,
                retriable: false,
                txnRequiresAbort: false,
            }

```